src/core.rs

```diff
@@ -911,7 +920,7 @@ pub trait WebauthnConfig {
     /// 15. If validation is successful, obtain a list of acceptable trust anchors (attestation
     /// root certificates or ECDAA-Issuer public keys) for that attestation type and attestation
     /// statement format fmt, from a trusted source or from policy. For example, the FIDO Metadata
-    /// Service [FIDOMetadataService] provides one way to obtain such information, using the
+    /// Service \[FIDOMetadataService\] provides one way to obtain such information, using the
```

Comment:
Doc generator yelled at me without escaping these

Reply:
Right--are you okay with escaping it like this?

Reply:
Yep